docs/book/_book/GLOSSARY.html

```diff
@@ -0,0 +1,468 @@
+
+<!DOCTYPE HTML>
+<html lang="" >
+    <head>
+        <meta charset="UTF-8">
+        <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
+        <title> · The Cluster API Book</title>
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="description" content="">
+        <meta name="generator" content="GitBook 3.2.3">
+        <meta name="author" content="SIG Lifecycle Team">
+        
```

Comment:
It is strange. Maybe there is a stray new line (or two!) somewhere. I'll look into it.

Reply:
Are you aware of this discussion on gitbook's future as an open source project:
https://github.com/GitbookIO/gitbook/issues/1808
https://github.com/GitbookIO/gitbook/issues/2148

it seems the new version does not support self-hosting anymore
https://docs.gitbook.com/v2-changes/important-differences

Reply:
It may be we that we have to or should migrate to netlify in the future. @timothysc suggested this last week since it is what k8s uses.

If we do it might also be nice to migrate kubebuilder as well. I am unsure if there is interest on that front but I think there is value in leveraging kubebuilder to the extent possible since this will make it easier for others to create high quality CRDs. @pwittrock 

For now I'd like to get this merged so that we can parallize and iterate. @detiber, @roberthbailey, thoughts?

Reply:
I see no reason to block this now, but it'll definitely be worth tracking it for later followup.

Reply:
+1 for merging, I didn't intend to block this, content is more important than the tool

k8s documentation is written in markdown, processed using Hugo and served using netlify
https://kubernetes.io/docs/contribute/start/#the-basics-about-our-docs

Reply:
Thanks for the link!